images/solution_architecture/high_level_architecture.svg

```diff
@@ -0,0 +1,3 @@
amily="Helvetica" font-size="12" font-weight="bold" x="2.2697309" y="25" textLength="34.669922">Certifi</tspan><tspan font-family="Helvetica" font-size="12" font-weight="bold" x="36.939653" y="25" textLength="64.67578">cate (DCC) </tspan><tspan font-family="Helvetica" font-size="12" font-weight="bold" x="31.592973" y="39" textLength="37.365234">Server</tspan></text><rect x="283.46457" y="342.99212" width="116.22047" height="45.35433" fill="white"/><rect x="283.46457" y="342.99212" width="116.22047" height="45.35433" stroke="black" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/><text transform="translate(288.46457 351.66929)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="bold" x="8.417853" y="11" textLength="92.71875">CovPass Issuer </tspan><tspan font-family="Helvetica" font-size="12" font-weight="bold" x="34.427619" y="25" textLength="37.365234">Server</tspan></text><text transform="translate(733.5236 353.68368)" fill="black"><tspan font-family="Helvetica Neue" font-size="12" font-weight="500" x="22.504" y="11" textLength="47.328">Retrieve </tspan><tspan font-family="Helvetica Neue" font-size="12" font-weight="500" x=".394" y="25.335999" textLength="65.1">digital certifi</tspan><tspan font-family="Helvetica Neue" font-size="12" font-weight="500" x="65.494" y="25.335999" textLength="23.112">cate</tspan></text><line x1="600.94488" y1="351.49606" x2="848.41514" y2="351.49606" stroke="blue" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/><circle cx="706.3796" cy="350.82852" r="12.000019" fill="white"/><circle cx="706.3796" cy="350.82852" r="12.000019" stroke="blue" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/><text transform="translate(703.1296 365.84252)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x=".16699219" y="11" textLength="8.6660156">R</tspan></text><path d="M 700.61616 369.64719 L 694.1299 372.84264 L 700.61616 376.03809 Z" fill="blue"/><path d="M 700.61616 369.64719 L 694.1299 372.84264 L 700.61616 376.03809 Z" stroke="blue" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/><path d="M 399.68504 367.50393 L 434.85433 367.50393 C 434.85433 361.00393 447.85433 361.00393 447.85433 367.50393 L 490.3937 367.50393" stroke="blue" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/><circle cx="469.88976" cy="367.8364" r="12.000019" fill="white"/><circle cx="469.88976" cy="367.8364" r="12.000019" stroke="blue" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/><text transform="translate(466.63976 382.8504)" fill="black"><tspan font-family="Helvetica" font-size="12" font-weight="500" x=".16699219" y="11" textLength="8.6660156">R</tspan></text><path d="M 464.12633 386.65506 L 457.6401 389.8505 L 464.12633 393.04596 Z" fill="blue"/><path d="M 464.12633 386.65506 L 457.6401 389.8505 L 464.12633 393.04596 Z" stroke="blue" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"/></g></g></svg>
```

Comment:
@tklingbeil we should also add the following items to the diagram:
- CWA is also storing DCCs
- CWA Server is replicating Digital Signing Certificates (DSCs) from CovPass to CDN for signature verification of DCCs
- CWA Server is replicating DCC business rules and booster notification rules from the National Rule Distribution Service (open source in our org) to CDN